index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   admission-controller:
   - apiVersion: v2
     appVersion: 0.0.1
-    created: "2022-05-05T22:09:37.397718-07:00"
```

Comment:
Looks like we updated time stamps of charts that are not in this PR?

Reply:
Yeah, sadly, `helm` doesn't have a flag to avoid updating these timestamps.